scripts/generate_package_diffs_for_flavors/generate_package_diffs_for_flavors.py

```diff
@@ -134,10 +135,23 @@ def compare_flavor(flavor_path_1: Path, working_copy_1: Path, working_copy_1_nam
 
 
 def get_last_git_tag() -> str:
+    working_copy_main = tempfile.mkdtemp()
+    checkout_git_tag_as_worktree(working_copy_main, "master")
+    old_dir = os. getcwd()
+    os.chdir(working_copy_main)
+    get_pull_command = ["git", "pull"]
+    pull_result = subprocess.run(get_pull_command, stderr=subprocess.PIPE)
+    pull_result.check_returncode()
     get_last_tag_command = ["git", "describe", "--abbrev=0", "--tags"]
```

Comment:
Why not 
```suggestion
    get_last_tag_command = ["git", "describe", "--abbrev=0", "--tags", "origin/master"]
```
before that fetch, instead of creating a worktree

Reply:
I'am ashamed, much easier....